src/d3a/d3a_core/simulation.py

```diff
@@ -142,7 +142,7 @@ def _init(self, slowdown, seed, paused, pause_after):
         # cli-call and behave integration tests
         # Furthermore, random.seed() is also called before this following block by setuptools
         if seed is not None:
-            random.seed(seed)
+            random.seed(int(seed))
```

Comment:
When I was testing the export in another branch, I saw that the seed is not reported, if not set explicitly.
So, can you please add an update of the `self.initial_params.seed` in order for the d3a to also export the seed, if it is determined by the simulation itself (else block)? Thanks a lot!

Reply:
I'll add it, thanks!

Reply:
@hannesdiedrich  btw, is the comment above still relevant? If it is not let me know to remove it, thanks!

Reply:
@spyrostz Not that I know of. This is probably solved by the changes you propose here. But I am not sure. IMO this comment does not help a lot. It just tells us that there might be an issue (experienced 1 year ago or so) but does not tell exactly what. So i propose to delete it.

Reply:
Thanks, I also think we should delete that since it is not clear enough, removing it